specs/phase0/beacon-chain.md

```diff
@@ -1092,7 +1095,7 @@ def initiate_validator_exit(state: BeaconState, index: ValidatorIndex) -> None:
     exit_queue_epoch = max(exit_epochs + [compute_activation_exit_epoch(get_current_epoch(state))])
     exit_queue_churn = len([v for v in state.validators if v.exit_epoch == exit_queue_epoch])
     if exit_queue_churn >= get_validator_churn_limit(state):
-        exit_queue_epoch += Epoch(1)
+        exit_queue_epoch = Epoch(exit_queue_epoch + 1)
```

Comment:
Also rather verbose. Maybe we should cast the above creation of exit epochs instead? Hmm

Reply:
Awww it seems mypy is happy with `exit_queue_epoch += Epoch(1)` now! Maybe it's due to `remerkleable` new release, or I got wrong with something. I'll change it back. Thanks!